frontend/src/schemas/SchemaSelect.tsx

```diff
@@ -56,6 +66,12 @@ export default function SchemaSelect({ entry }: SchemaSelectProps) {
     [currentUser, entry, mutateEntry, router],
   )
 
+  let selectionCallback = entryCallback
```

Comment:
The references should still be stable with how this has been implemented, but to ensure they are we could do the following:
```
  const selectionCallback = useMemo(
    () => (schemaKind === SchemaKind.ACCESS_REQUEST ? accessRequestCallback : entryCallback),
    [schemaKind, accessRequestCallback, entryCallback],
  )
```